### DISCUSSION TITLE
ILI9488 with an ESP32-S3

### DISCUSSION THREAD

**Discussion by builderdev212** [Bodmer/TFT_eSPI#2934] [Q&A - General]
I'm working on a project with an ILI9488 and an ESP32-S3. From reading other discussions here, it sounds like the ESP32 SPI _with_ DMA is faster than 8-bit parallel... and in this case, the ESP32-S3 doesn't have DMA support. So in terms of speed, would using 8/16-bit parallel be faster? (and I guess on that topic) does the ESP32-S3 support 16-bit parallel?

**Comment by Bodmer** [comment]
The ILI9488 is supported in SPI and 8 bit parallel modes, both without DMA. A full list of supported displays and transfer options in in the ReadMe.

I suspect that 8 bit parallel mode will have a faster pixel write speed with the ILI9488
